src/test/java/org/cactoos/io/HeadInputStreamTest.java

```diff
@@ -44,7 +44,13 @@ public void testSkippingLessThanTotal() throws Exception {
             new InputOf("testSkippingLessThanTotal").stream(),
             5
         );
-        stream.skip(3L);
+        final long skipped = stream.skip(3L);
+        MatcherAssert.assertThat(
+            skipped,
```

Comment:
@ilyakharlamov move  the line with `stream.skip(3L)` to `skipped` place.

Reply:
done